javascript/javascript3/week1/weather/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Fetch Weather API</title>
+  </head>
+
+  <body>
+    <h1>Weather Forecast</h1>
+    <input type="text" id="userinput" placeholder="Enter city name" />
+    <button id="button1">SEARCH</button>
```

Comment:
button type should be submit.

Reply:
could you explain why it has to be a submit